packages/cli/src/verifyContract.ts

```diff
@@ -0,0 +1,43 @@
+import { forge } from "@latticexyz/common/foundry";
+import { Address, ByteArray, Hex, getCreate2Address } from "viem";
+import { salt as defaultSalt } from "./deploy/common";
+
+type VerificationOptions = {
+  address: Hex;
+  name: string;
+  rpc: string;
+  verifier?: string;
+  verifierUrl?: string;
+};
+
+type ForgeOptions = { profile?: string; silent?: boolean; env?: NodeJS.ProcessEnv; cwd?: string };
+
+type Create2Options = {
+  bytecode: ByteArray | Hex;
+  from: Address;
+  salt?: ByteArray | Hex;
+};
+
+export async function verifyContract(
+  { address, name, rpc, verifier, verifierUrl }: VerificationOptions,
+  options?: ForgeOptions,
+) {
+  const args = ["verify-contract", address, name, "--rpc-url", rpc];
+
+  if (verifier) {
+    args.push("--verifier", verifier);
+  }
+  if (verifierUrl) {
+    args.push("--verifier-url", verifierUrl);
+  }
+  await forge(args, options);
+}
+
+export async function verifyContractCreate2(
```

Comment:
to match `getCreate2Address`
```suggestion
export async function verifyCreate2Contract(
```

Reply:
This function is gone!